packages/openapi-spec-types/src/v3/openapi-v3-spec-types.ts

```diff
@@ -0,0 +1,395 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-spec-types
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/*
+ * OpenApiSpec - A typescript representation of OpenApi 3.0.0
+ */
+
+// tslint:disable:max-line-length
+
+import * as OAS3 from 'openapi3-ts';
+// Export spec interfaces from the community module if missing in our package
+export * from 'openapi3-ts';
+
+export type OpenApiSpec = OAS3.OpenAPIObject;
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
+// tslint:disable-next-line:no-any
+export type ExtensionValue = any;
+
+/**
+ * The location of a parameter.
+ * Possible values are "query", "header", "path" or "body".
+ * <p>Specification:
+ * https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md
+ * search "The location of the parameter"
+ */
+export type ParameterLocation = 'query' | 'header' | 'path' | 'cookie';
```

Comment:
Ideally, I think this should come from `opeanpi3-ts` package. 

If you agree then please add a code comment here, ideally pointing to a GitHub issue tracking this problem (maybe in https://github.com/metadevpro/openapi3-ts?).

https://github.com/metadevpro/openapi3-ts/blob/121b7b4d345e3240bad47cb947e4a9ad569f1e00/src/model/OpenApi.ts#L110-L112
```ts

export interface ParameterObject extends ISpecificationExtension {
    name: string;
    in: string; // "query" | "header" | "path" | "cookie";
```

Reply:
@bajtos My plan is:
- important interfaces(like `ParameterObject`): for a better control, I suggest we override it and the sub interfaces its properties uses.
- interfaces we haven't used yet(like `SecurityObject`): we import them from the community module `openapi3-ts`

I also talked with Raymond about this yesterday, will add more context about it in the "type" PR.